backend/src/db/reports.js

```diff
@@ -62,4 +66,5 @@ const RAW_REPOT_QUERY =
       )
       as courseDetails 
       ON "courseId" = courseDetails.id
-    `;
\ No newline at end of file
+      ORDER BY start_date DESC
```

Comment:
isn't the actual field value `start` instead of `start_date` ?. Or am I wrong?.

Reply:
The initial field name is start, however we rename it to start_date.
I believe as order by will be carried out after the select, we should refer to this value using the renamed value from the select clause.

Reply:
Thank you for the clarification. 👍